# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xosview/libx11.patch
# Copyright (C) 2012 - 2013 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Let it find libX11, when not in /usr/lib, and especially lib64.

	- Rene Rebe <rene@exactcode.de>

--- xosview-1.16/Makefile.vanilla	2013-11-19 18:10:31.000000000 +0100
+++ xosview-1.16/Makefile	2013-11-19 18:10:42.000000000 +0100
@@ -18,7 +18,7 @@
 # Required build arguments
 
 CPPFLAGS += $(OPTFLAGS) -I. -MMD
-LDLIBS += -lX11 -lXpm
+LDLIBS += $(shell pkg-config x11 --libs) -lXpm
 
 OBJS = Host.o \
 	Xrm.o \
